File: csharp/DVSClient/DVSClient/server/address/layout/RestApiCreateLayoutResponse.cs
using Newtonsoft.Json;

namespace DVSClient.Server.Address.Layout
{
    public class RestApiCreateLayoutResponse : RestApiResponse
    {
        [JsonProperty("error")]
        public RestApiResponseError? Error { get; set; }

        [JsonProperty("result")]
        public RestApiCreateLayoutResult? Result { get; set; }
    }
}

File: typescript/src/server/address/layout/restApiCreateLayoutResponse.ts
import { RestApiResponse } from "../../restApiResponse";
import { RestApiResponseError } from "../../restApiResponseError";
import { RestApiCreateLayoutResult } from "./restApiCreateLayoutResult";

export type RestApiCreateLayoutResponse = RestApiResponse & {
    error?: RestApiResponseError;
    result?: RestApiCreateLayoutResult;
};